### PR TITLE
chore: use `prettier-plugin-tailwindcss`

### PR DESCRIPTION
**Pull request by MichaelDeBoey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "msw": "^0.39.1",
     "pm2": "^5.2.0",
     "prettier": "^2.5.1",
+    "prettier-plugin-tailwindcss": "^0.1.8",
```
Comment (kentcdodds) [inline package.json:71]: Is this still an issue with tailwind 3? https://github.com/tqwewe/prettier-plugin-tailwind/issues/29

**Review by kentcdodds — COMMENTED** [review]
Thanks for this! One question.

File: package.json (hunk 1 above)
Comment (MichaelDeBoey) [reply package.json:71]: This is the official plugin by the Tailwind team.

`prettier-plugin-tailwind` (without the `css` at the end), is deprecated now
https://github.com/tqwewe/prettier-plugin-tailwind/issues/46#issuecomment-1020509621

Comment (kentcdodds) [reply package.json:71]: Nice! Thanks!

**Review by kentcdodds — APPROVED** [review]

**Comment by sun33t** [comment]
@MichaelDeBoey I know this is an old thread but I've just come across the repo in my recent remix journey and thought I'd point out that the prettier config doesn't make use of the tailwindcss plugin. It's a tiny thing but I thought I'd point it out just in case it's been missed 👍

**Comment by MichaelDeBoey** [comment]
@sun33t This will be fixed by #243

**Pull request merged**